### PR TITLE
added wallet options

### PR DESCRIPTION
**Pull request by unicorn-talent**
Portis
WalletConnect

### CODE REVIEW

File: src/pages/SignIn.tsx [hunk 1]
```diff
@@ -33,6 +35,18 @@ export default function SignIn(): ReactElement {
             key: VITE_FORT_MATIC_KEY, // required
           },
         },
+        portis: {
+          package: Portis, // required
+          options: {
+            id: "a180917a-e8d2-43a9-a782-456579bf39b9", // required
```
Comment (kmozurkewich) [inline src/pages/SignIn.tsx:41]: @unicorn-talent  - what is this ID used for relative to Portis?

File: src/pages/SignIn.tsx [hunk 2]
```diff
@@ -33,6 +35,18 @@ export default function SignIn(): ReactElement {
...
+          },
+        },
+        walletconnect: {
+          package: WalletConnectProvider,
+          options: {
+            infuraId: "8cf3cad623da43f9a84ab5ac94230cf6",
```
Comment (kmozurkewich) [inline src/pages/SignIn.tsx:47]: @unicorn-talent  - similar question here for infura.  Imagine it's a working client id?  If so, should be in a config file.

**Review by kmozurkewich — COMMENTED** [review]
@unicorn-talent - please comment on comments.

File: src/pages/SignIn.tsx (hunk 1 above)
Comment (unicorn-talent) [reply src/pages/SignIn.tsx:41]: we need to register ID in Portis

File: src/pages/SignIn.tsx (hunk 2 above)
Comment (unicorn-talent) [reply src/pages/SignIn.tsx:47]: yes, those should be in env file

**Pull request closed without merging**